x-pack/plugins/secops/public/components/notes/translations.ts

```diff
@@ -15,21 +15,33 @@ export const USER = i18n.translate('xpack.secops.notes.user', {
 });
 
 export const ADD_NOTE = i18n.translate('xpack.secops.notes.addNote', {
-  defaultMessage: 'Add note',
+  defaultMessage: 'Add Note',
 });
 
 export const ADD_A_NOTE = i18n.translate('xpack.secops.notes.addANote', {
   defaultMessage: 'Add a Note',
 });
 
+export const ADDED_A_NOTE = i18n.translate('xpack.secops.notes.addedANote', {
```

Comment:
Should these i18n message ids end with a type as per the [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#naming-convention)? 

e.g. `xpack.secops.notes.addedANoteAction` or whichever type is most relevant.